docker/CMSRucioClient/scripts/CMSRSE.py

```diff
@@ -310,12 +313,20 @@ def _get_protocol(self, proto_json, protos_json):
             for method, weight in domains['wan'].items():
                 if weight and protocol_name in PROTO_WEIGHT_TPC:
                     if method.startswith("third_party_copy"):
-                        domains['wan'][method] = PROTO_WEIGHT_TPC[protocol_name]
+                        if access == 'global-ro' and method != 'third_party_copy_read':
+                            domains['wan'][method] = 0
+                        else:
+                            domains['wan'][method] = PROTO_WEIGHT_TPC[protocol_name]
+                            
                     else:
-                        domains['wan'][method] = PROTO_WEIGHT_RWD[protocol_name]
+                        if access == 'global-ro' and method != 'read':
+                            domains['wan'][method] = 0
+                        else:
+                            domains['wan'][method] = PROTO_WEIGHT_RWD[protocol_name]
         except KeyError:
-            pass  # We're trying to modify an unknown protocol somehow
-        # TODO: Make sure global-rw is set
+            # We're trying to modify an unknown protocol somehow
+            logging.error("Unknown protocol: "+protocol_name)
```

Comment:
This should never happen, since we removed returned in line 301 for unknown protocols. 
But if this is happening, it should reflect in our logs.